doc/user-guide/jupyter/ipyvtk_plotting.rst

```diff
@@ -61,18 +55,17 @@ Where environment.yml is:
       - conda-forge
       - defaults
     dependencies:
-      - jupyterlab=2.2.9
-      - itkwidgets=0.32.0
-      - ipywidgets=7.5.1
```

Comment:
unless I am mistaken, by default pyvista does not include ipyvtklink. So we should add it here.

Reply:
Good catch 🤦🏻‍♂️